src/xbps/repositories/mirrors/index.md

```diff
@@ -53,3 +53,21 @@ sub-repository.
 
 Void Linux is also mirrored on the Tor network. See [Using Tor
 Mirrors](./tor.md) for more information.
+
+## Creating a mirror
+
+If you'd like to set up a mirror, and are confident you can keep it reasonably
+up-to-date, follow one of the many guides available for mirroring with
+[rsync(1)](https://man.voidlinux.org/rsync.1), then submit a pull request to
+[the void-docs repository](https://github.com/void-linux/void-docs) to add your
+mirror to this page.
+
+If your mirror is going to be exposed outside of your region, this should be
+noted in the documentation. If your mirror is internal to your region, i.e.
+behind some kind of carrier NAT, this should also be noted.
+
+A full mirror requires around 1TB of storage.
+
+Please keep in mind that we pay bandwidth for all data sent out from the Tier 1
+mirrors. You can respect this by only mirroring if your use case for your mirror
+will offset the network throughput consumed by your mirror syncing.
```

Comment:
"It is also possible to mirror only part of the repositories. Excluding debug packages is one way of decreasing the load on the Tier 1 mirros that has low impact on users."

Reply:
Added to the previous para, where it seems to fit best, with some small changes.